docs/src/tutorials/tutorial_4.md

```diff
@@ -62,7 +62,7 @@ Although one could use standard an `Action` to achieve the same functionality, `
 
 The `subactions` keyword uses a list of [`SubAction`](@ref) structs which are defined in a similar fashion as `Action` with `frames` and a `function` but are in some sense simpler than the `Action`. 
 
-A function of a `SubAction` is normally either [`appear`](@ref) or [`disappear`](@ref) at the moment or one of two transformations: [`Translation`](@ref) and [`Rotation`](@ref).
+A function of a `SubAction` is normally either [`appear`](@ref) or [`disappear`](@ref) at the moment or one of these transformations: [`Translation`](@ref), [`Rotation`](@ref) and [`Scaling`](@ref).
```

Comment:
Do you think we should include scaling in a separate tutorial since you never actually use scaling in the code of this tutorial?

Reply:
Yes that would be nice. This PR is actually more complex than anticipated. I will add a tutorial in the next days once the functionality is good and approved.

Reply:
Can you open an issue on this new tutorial and resolve this convo? Feel free to assign me the tut.